HostAbroadTests/.classpath

```diff
@@ -18,5 +18,16 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
```

Comment:
Pls add only the two classes that you created for this task. Everything else seems ok.